src/averageAges.js

```diff
@@ -37,7 +42,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter((person) => person.sex === 'f');
+
+  women = withChildren !== undefined
```

Comment:
No need to explicitly write what **withChildren**  should not be equal undefined.
```suggestion
  women = withChildren
```